src/ImReadyManager.ts

```diff
@@ -66,7 +66,7 @@ class ImReadyManager extends Component<ImReadyEvents> {
 
       loader.check();
       loader.on("error", e => {
-        this.onError(index, e.target);
+        this.elementInfos[index] && this.onError(index, e.target);
```

Comment:
This code would be better checked if it was in `onError`.

Reply:
I changed this part and modified the test together, checks whether the error event is valid than the number of times `onError` is called.